### ISSUE TITLE
Incorrect flood table sizes with stacking

### ISSUE THREAD

**Issue by mab68** [faucetsdn/faucet#3509]
Faucet allocates the flood table sizes to 32

Switches 1-5 needs 44 flood table rules
Switches 6-12 needs 34 flood table rules

![topology](https://user-images.githubusercontent.com/33644913/78842722-f1281980-7a54-11ea-9b68-c83ae00b09a0.png)

CONFIG: [faucet.yaml.txt](https://github.com/faucetsdn/faucet/files/4453236/faucet.yaml.txt)

faucet-1 flowdump: 
[faucet-1 flowdump.txt](https://github.com/faucetsdn/faucet/files/4453277/faucet-1.flowdump.txt)

```
m@b68:/var/tmp$ cat 'faucet-1 flowdump' | grep table=3 | wc -l
44
```

Faucet-6 flowdump: 
[faucet-6 flowdump.txt](https://github.com/faucetsdn/faucet/files/4453283/faucet-6.flowdump.txt)

```
m@b68:/var/tmp$ cat 'faucet-6 flowdump' | grep table=3 | wc -l
34
```

Reference: #3198

**Comment by mab68** [comment]
The correct formula for flood table sizes seems to be:
`(len(self.vlans) * (len(self.stack_ports) + 1) * 5) + 4`

**Comment by mab68** [comment]
Fixed in #3597
